src/site/es/markdown/index.md

```diff
@@ -0,0 +1,18 @@
+title: MyBatis 3 | Introducción
+author: Clinton Begin,Eduardo Macarron
+
+## Introducción
+
+### ¿Qué es MyBatis?
+
+MyBatis es un framework de persistencia que soporta SQL, procedimientos almacenados y mapeos avanzados. MyBatis elimina casi todo el código JDBC, el establecimiento manual de parámetros y la obtención de resultados. MyBatis puede configurarse con XML o anotaciones y permite mapear typos de datos primitivos, objetos de tipo Map y POJOs (Plain Old Java Objects) a registros de base de datos.
+
+If you are completely new to database access in Java, https://www.marcobehler.com/guides/a-guide-to-accessing-databases-in-java contains a good overview of the various parts, pieces and options and shows you where exactly MyBatis fits in.
```

Comment:
Please remove this new sentence.
The linked site is written in English, so it may not be appropriate in non-English docs.

If you want to make a new change, please do it in a separate PR after the conversion.